plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -339,7 +339,7 @@ public boolean isExtendedStatisticsEnabled()
     }
 
     @Config(EXTENDED_STATISTICS_ENABLED)
-    @ConfigDescription("Use extended statistics collected by ANALYZE")
+    @ConfigDescription("Allow ANALYZE and use of extended statistics collected by it")
```

Comment:
Maybe 
```
Enable collection (ANALYZE) and use of extended statistics.
```

Reply:
this is better, thanks, updating in both Iceberg and Delta